storage-access.bs

```diff
@@ -129,6 +129,14 @@ A {{Document}} is in a <dfn>first-party-site context</dfn> if it is the [=active
 
 A {{Document}} is in a <dfn>third party context</dfn> if it is not in a [=first-party-site context=].
 
+Let <dfn export abstract-op>whether the user agent explicitly allows unpartitioned cookie access</dfn> be an algorithm that, given a [=tuple=] |tuple| consisting of two [=sites=], runs the following steps. This algorithm returns "`none`", "`allow`" or "`disallow`".
+        
+    Note: A user agent's settings might explicitly allow or disallow unpartitioned cookie access through per-site allow-lists, the user changing global browser settings, or similar custom overrides.
+        
+    1. If the user agent does not have explicit settings for unpartitioned cookie access for |tuple|, return "`none`".
+    1. If the user agent's settings explicitly allow unpartitioned cookie access for |tuple|, return "`allow`".
+    1. If the user agent's settings explicitly disallow unpartitioned cookie access for |tuple|, return "`disallow`".
```

Comment:
This should be an assert followed by returning disallow.

Reply:
Done